### ISSUE TITLE
Color contrast in the alert component is a tad bit too low to be easily readable

### ISSUE THREAD

**Issue by hampusn** [sitevision/envision#71]
The contrast on the [alert component](http://envisionui.io/components/alert/) as seen in the attachment is a tad bit low and does not meet any WCAG level. This might already exist in the pipeline but I thought I should mention it anyways.

![env-alert-variations](https://user-images.githubusercontent.com/588352/32718254-d41b7668-c85c-11e7-9807-48feca2ba63d.png)

Links to webaim.org's contrast checker for each variation:
* [Contrast for Success](https://webaim.org/resources/contrastchecker/?fcolor=878787&bcolor=C5EBC5)
* [Contrast for Warning](https://webaim.org/resources/contrastchecker/?fcolor=878787&bcolor=F4DEA9)
* [Contrast for Danger](https://webaim.org/resources/contrastchecker/?fcolor=878787&bcolor=ED7E7E)
* [Contrast for Info](https://webaim.org/resources/contrastchecker/?fcolor=878787&bcolor=BBEBF0)

**Comment by robinlagren** [comment]
Thanks for pointing this out. Will look into it!

**Comment by melineric** [comment]
I've updated the Alert component.

Thank you Hampus!
